### PR TITLE
Conserwa Test Decoders

### PR DESCRIPTION
**Pull request by nikolaskyr**

### CODE REVIEW

File: manufacturers/Conserwa/SensecapS2120/uplink_decoder.js [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+					ngsild_payload[0].airTemperature = ngsildInstance(messages[i].measurementValue, time, 'CEL', 'Raw');
+				}
+                else if (messages[i].measurementId === 1) {
+					ngsild_payload[0].airHumidity = ngsildInstance(messages[i].measurementValue, time, 'P1', 'Raw');
+                }
+				else if (messages[i].measurementId === 2) {
+					ngsild_payload[0].lightIntensity = ngsildInstance(messages[i].measurementValue, time, 'LUX', 'Raw');
+                }
+				else if (messages[i].measurementId === 3) {
+					ngsild_payload[0].uvIndex = ngsildInstance(messages[i].measurementValue, time, '', 'Raw');
```
Comment (gasserluc) [inline manufacturers/Conserwa/SensecapS2120/uplink_decoder.js:147]: If there is no unitCode, use `null` instead of `''`

**Comment by gasserluc** [comment]
The file paths are wrong, please move the config and decoder to `manufacturers/SenseCAP/models/S2104/` and to `manufacturers/SenseCAP/models/S2120/`
Other than that (and the empy unitCode), the files are correct.

**Review by gasserluc — CHANGES_REQUESTED** [review]
The file paths are wrong, please move the config and decoder to manufacturers/SenseCAP/models/S2104/ and to manufacturers/SenseCAP/models/S2120/
Other than that (and the empy unitCode), the files are correct.

File: manufacturers/Conserwa/SensecapS2120/uplink_decoder.js (hunk 1 above)
Comment (nikolaskyr) [reply manufacturers/Conserwa/SensecapS2120/uplink_decoder.js]: ok done

**Review by gasserluc — APPROVED** [review]

**Pull request merged**